javascript/grid-ui/BUILD.bazel

```diff
@@ -78,3 +79,18 @@ zip_file(
         "//java/server/src/org/openqa/selenium/grid:__pkg__",
     ],
 )
+
+jest_test(
+    name = "test",
+    args = [
+        "--no-cache",
+        "--no-watchman",
+        "--ci",
+        "--colors",
+        "--config",
+        "jest.config.js"
+    ],
+    data = [
+        "//:jest.config.js"
```

Comment:
By having `//` you're telling it to look at the root of the repo which means it will error.